library/alloc/src/collections/linked_list.rs

```diff
@@ -407,15 +407,24 @@ impl<T> Default for LinkedList<T> {
 }
 
 impl<T> LinkedList<T> {
-    /// Creates an empty `LinkedList`.
+    /// A doubly-linked list with owned nodes.
```

Comment:
It looks like you changed the definition for `new`, and then also changed the documentation for `new` to use a completely different function?